Client/Application/AppDelegate.swift

```diff
@@ -408,6 +408,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
     }
 
     func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([Any]?) -> Void) -> Bool {
+        if userActivity.userInfo?["action"] as? String == "newTab" {
```

Comment:
It might be better to check on userActivity.activityType. You won't have to cast it, and it won't be optional.